menus/glsl-preview.cson

```diff
@@ -16,4 +16,5 @@
   '.tree-view .file':[
     {label: 'Glsl Preview: Add texture', command: 'glsl-preview:addTextureTreeView'}
     {label: 'Glsl Preview: Remove texture', command: 'glsl-preview:removeTexture'}
+    {label: 'Glsl Preview: Toggle texture button', command: 'glsl-preview:toggleTextureButton'}
```

Comment:
This adds the menu entry for when you right click on files in the tree view. Might make more sense to put it on the preview itself, using the selector '.glsl-preview-pane'.